### PR TITLE
Use cItem.m_LoreTable in give command

### PR DESCRIPTION
**Pull request by peterbell10**
Fixes the warnings since cuberite/cuberite#3882 was merged.

### CODE REVIEW

File: give.lua [hunk 1]
```diff
@@ -281,11 +281,8 @@ local function GiveItemCommand( Split, Player, SafeCommand )
 
 			-- Set Lore if given
 			if DataTagTable.display.Lore then
-				local Lore = ""
-				for _, value in ipairs(DataTagTable.display.Lore) do
-					Lore = Lore .. value .. "`"  -- Newline is '`' character rather than "\n"
-				end
-				Item.m_Lore = Lore
+				local Lore = DataTagTable.display.Lore
```
Comment (bearbin) [inline give.lua]: What is the point of this Lore variable? Why not directly assign it as:

    Item.m_LoreTable = DataTagTable.display.Lore

Comment (peterbell10) [inline give.lua]: This way it matches the custom name code above it.  Would you prefer both be assigned directly?

Comment (peterbell10) [inline give.lua]: I've changed both now.

**Comment by peterbell10** [comment]
Must admit I'd missed the other use of `Name`, still works without but maybe it was cleaner before?

**Pull request merged**